examples/lang/erc20/src/lib.rs

```diff
@@ -68,14 +67,14 @@ contract! {
         /// Returns the total number of tokens in existence.
         pub(external) fn total_supply(&self) -> Balance {
             let total_supply = *self.total_supply;
-            env.println(&format!("Erc20::total_supply = {:?}", total_supply));
+            env::println(&format!("Erc20::total_supply = {:?}", total_supply));
```

Comment:
Are we able to still use `env.println` ? How about other `env.*` syntax elements?
If this patch introduces a shift here we need to make sure how to handle future cases.

Reply:
Yes you can still do both `env.println` and `env::println`. In this case I was testing that `env::println` still works.

Reply:
See also: https://github.com/paritytech/ink/issues/125